editor/scripts/bundle.py

```diff
@@ -312,6 +312,41 @@ def get_exe_suffix(platform):
     return ".exe" if 'win32' in platform else ""
 
 
+def strip_platform_files(platform, jar):
+    zin = zipfile.ZipFile(jar, 'r')
+    files = zin.namelist()
+    files_to_remove = []
+
+    # find files to remove from libexec/* except dmengine and except in platform folder
+    libexec_platform = "libexec/" + platform
+    for file in files:
+        if file.startswith("libexec") and not file.startswith(libexec_platform) and not file.endswith("/") and not "dmengine" in file:
+            files_to_remove.append(file)
+
+    # find libs to remove in the root folder
+    for file in files:
+        if "/" not in file:
+            if platform == "x86_64-darwin" and (file.endswith(".so") or file.endswith(".dll")):
+                files_to_remove.append(file)
+            elif platform == "x86_64-win32" and (file.endswith(".so") or file.endswith(".dylib")):
+                files_to_remove.append(file)
+            elif platform == "x86_64-linux" and (file.endswith(".dll") or file.endswith(".dylib")):
+                files_to_remove.append(file)
```

Comment:
Howcome the files were included in the first place?
Shouldn't the fix be there?

Reply:
I think it's specified here: 
https://github.com/defold/defold/blob/c5a77bd14529894bccd84fa73264111bbca4c4aa/editor/project.clj#L81-L119

not sure how to specify only needed libs depends on platform there

Reply:
We take everything from bob and the editor and put it into editor/target/defold-editor-2.0.0-SNAPSHOT-standalone.jar. We then use this jar file for all editor builds regardless of platform. So the first shot at something platform specific is when we actually bundle the editor for a specific platform.

We could obviously create on editor/target/defold-editor-2.0.0-SNAPSHOT-standalone.jar per platform if we want to, but I don't see that as better than the current solution.

Reply:
> I think it's specified here

You're right. We might be able to exclude something there.